swagger_server/swagger/swagger.yaml

```diff
@@ -1015,6 +1015,34 @@ components:
         - - treatment: Tourniquet
           - location: right forearm
         scenario_id: scenario_id
+    ProbeOption:
```

Comment:
I don't think this should go in the swagger file.  This file is for the TA2-TA3 API, which doesn't involve ProbeOptions.  We just need the ProbeOptions (and Probe) model object for use with TA1.